docs/bugs_and_glitches.md

```diff
@@ -246,7 +246,34 @@ DefenseDownHit:
 
 ([Video](https://www.youtube.com/watch?v=uRYyzKRatFk))
 
-*To do:* Identify specific code causing this bug and fix it.
+This is a bug with `BattleCommand_Counter` in [engine/battle/move_effects/counter.asm](/engine/battle/move_effects/counter.asm) and `BattleCommand_MirrorCoat` in [engine/battle/move_effects/mirror_coat.asm](/engine/battle/move_effects/mirror_coat.asm):
+
+```asm
+	; BUG: Move should fail with all non-damaging battle actions
+	ld hl, wCurDamage
+	ld a, [hli]
+	or [hl]
+	ret z
+```
+
+**Fix:**
+
+```asm
+	ld hl, wCurDamage
+	ld a, [hli]
+	or [hl]
+	jp z, .failed
+```
+
+Add this to the end of each file:
+
+```asm
+	.failed
```

Comment:
This shouldn't be indented. Otherwise these fixes look OK.

Reply:
Fixed in latest commit.